docs/delete_proxy.md

```diff
@@ -0,0 +1,28 @@
+# delete_proxy
+
+Delete the proxy configuration from the Rubrik cluster.
+```py
+def delete_proxy(self, timeout=15)
+```
+
+## Keyword Arguments
+| Name        | Type | Description                                                                 | Choices | Default |
+|-------------|------|-----------------------------------------------------------------------------|---------|---------|
+| timeout  | int  | The number of seconds to wait to establish a connection the Rubrik cluster before returning a timeout error.  |         |    15     |
+
+## Returns
+| Type | Return Value                                                                                   |
+|------|-----------------------------------------------------------------------------------------------|
+| str  | No change required. The proxy configuration is already cleared out. |
+| dict  | The full API response for `DELETE /node_management/proxy_config`. |
```

Comment:
`The full API response for `DELETE /node_management/proxy_config` ... which version of the API?